content/docs/quickstart/sqlalchemy.md

```diff
@@ -2,13 +2,23 @@
 title: Run a SQL Alchemy App
 ---
 
+### Introduction
+
 SQLAlchemy is among the most popular ORMs in the Python universe. Because Neon is fully compatible with vanilla PostgreSQL, you only need to fill in the correct connection details.
 
-Prerequisites:
+## Prerequisites
+
+To complete this section, you will need to already have the following:
+
+- Created a Project on Neon Cloud service
+- Have SQLAlchemy installed with a PostgresSQL driver (such as psycopg2, the default driver for PostgreSQL in SQLAlchemy)
+  - Use the following guide for installation details on [SQLAlchemy](https://docs.sqlalchemy.org/en/14/intro.html#installation) and [psycopg2](https://www.psycopg.org/docs/install.html).
```

Comment:
I suspect "the following guide" can be read as if it refers to the guide below this line, not the hyperlinks